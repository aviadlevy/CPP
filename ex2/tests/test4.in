Vince Chase
Eric Murphy
G2G4
E7E6
F2F4
D8H4
