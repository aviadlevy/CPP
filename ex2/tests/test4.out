Enter white player name:
Enter black player name:
[2J  ABCDEFGH

8 [30;46m‚ôú[0m[30;42m‚ôû[0m[30;46m‚ôù[0m[30;42m‚ôõ[0m[30;46m‚ôö[0m[30;42m‚ôù[0m[30;46m‚ôû[0m[30;42m‚ôú[0m 8
7 [30;42m‚ôü[0m[30;46m‚ôü[0m[30;42m‚ôü[0m[30;46m‚ôü[0m[30;42m‚ôü[0m[30;46m‚ôü[0m[30;42m‚ôü[0m[30;46m‚ôü[0m 7
6 [0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m 6
5 [0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m 5
4 [0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m 4
3 [0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m 3
2 [37;46m‚ôü[0m[37;42m‚ôü[0m[37;46m‚ôü[0m[37;42m‚ôü[0m[37;46m‚ôü[0m[37;42m‚ôü[0m[37;46m‚ôü[0m[37;42m‚ôü[0m 2
1 [37;42m‚ôú[0m[37;46m‚ôû[0m[37;42m‚ôù[0m[37;46m‚ôõ[0m[37;42m‚ôö[0m[37;46m‚ôù[0m[37;42m‚ôû[0m[37;46m‚ôú[0m 1

  ABCDEFGH

Vince Chase: Please enter your move:
[2J  ABCDEFGH

8 [30;46m‚ôú[0m[30;42m‚ôû[0m[30;46m‚ôù[0m[30;42m‚ôõ[0m[30;46m‚ôö[0m[30;42m‚ôù[0m[30;46m‚ôû[0m[30;42m‚ôú[0m 8
7 [30;42m‚ôü[0m[30;46m‚ôü[0m[30;42m‚ôü[0m[30;46m‚ôü[0m[30;42m‚ôü[0m[30;46m‚ôü[0m[30;42m‚ôü[0m[30;46m‚ôü[0m 7
6 [0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m 6
5 [0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m 5
4 [0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[37;46m‚ôü[0m[0;42m [0m 4
3 [0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m 3
2 [37;46m‚ôü[0m[37;42m‚ôü[0m[37;46m‚ôü[0m[37;42m‚ôü[0m[37;46m‚ôü[0m[37;42m‚ôü[0m[0;46m [0m[37;42m‚ôü[0m 2
1 [37;42m‚ôú[0m[37;46m‚ôû[0m[37;42m‚ôù[0m[37;46m‚ôõ[0m[37;42m‚ôö[0m[37;46m‚ôù[0m[37;42m‚ôû[0m[37;46m‚ôú[0m 1

  ABCDEFGH

Eric Murphy: Please enter your move:
[2J  ABCDEFGH

8 [30;46m‚ôú[0m[30;42m‚ôû[0m[30;46m‚ôù[0m[30;42m‚ôõ[0m[30;46m‚ôö[0m[30;42m‚ôù[0m[30;46m‚ôû[0m[30;42m‚ôú[0m 8
7 [30;42m‚ôü[0m[30;46m‚ôü[0m[30;42m‚ôü[0m[30;46m‚ôü[0m[0;42m [0m[30;46m‚ôü[0m[30;42m‚ôü[0m[30;46m‚ôü[0m 7
6 [0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[30;46m‚ôü[0m[0;42m [0m[0;46m [0m[0;42m [0m 6
5 [0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m 5
4 [0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[37;46m‚ôü[0m[0;42m [0m 4
3 [0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m 3
2 [37;46m‚ôü[0m[37;42m‚ôü[0m[37;46m‚ôü[0m[37;42m‚ôü[0m[37;46m‚ôü[0m[37;42m‚ôü[0m[0;46m [0m[37;42m‚ôü[0m 2
1 [37;42m‚ôú[0m[37;46m‚ôû[0m[37;42m‚ôù[0m[37;46m‚ôõ[0m[37;42m‚ôö[0m[37;46m‚ôù[0m[37;42m‚ôû[0m[37;46m‚ôú[0m 1

  ABCDEFGH

Vince Chase: Please enter your move:
[2J  ABCDEFGH

8 [30;46m‚ôú[0m[30;42m‚ôû[0m[30;46m‚ôù[0m[30;42m‚ôõ[0m[30;46m‚ôö[0m[30;42m‚ôù[0m[30;46m‚ôû[0m[30;42m‚ôú[0m 8
7 [30;42m‚ôü[0m[30;46m‚ôü[0m[30;42m‚ôü[0m[30;46m‚ôü[0m[0;42m [0m[30;46m‚ôü[0m[30;42m‚ôü[0m[30;46m‚ôü[0m 7
6 [0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[30;46m‚ôü[0m[0;42m [0m[0;46m [0m[0;42m [0m 6
5 [0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m 5
4 [0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[37;42m‚ôü[0m[37;46m‚ôü[0m[0;42m [0m 4
3 [0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m 3
2 [37;46m‚ôü[0m[37;42m‚ôü[0m[37;46m‚ôü[0m[37;42m‚ôü[0m[37;46m‚ôü[0m[0;42m [0m[0;46m [0m[37;42m‚ôü[0m 2
1 [37;42m‚ôú[0m[37;46m‚ôû[0m[37;42m‚ôù[0m[37;46m‚ôõ[0m[37;42m‚ôö[0m[37;46m‚ôù[0m[37;42m‚ôû[0m[37;46m‚ôú[0m 1

  ABCDEFGH

Eric Murphy: Please enter your move:
[2J  ABCDEFGH

8 [30;46m‚ôú[0m[30;42m‚ôû[0m[30;46m‚ôù[0m[0;42m [0m[30;46m‚ôö[0m[30;42m‚ôù[0m[30;46m‚ôû[0m[30;42m‚ôú[0m 8
7 [30;42m‚ôü[0m[30;46m‚ôü[0m[30;42m‚ôü[0m[30;46m‚ôü[0m[0;42m [0m[30;46m‚ôü[0m[30;42m‚ôü[0m[30;46m‚ôü[0m 7
6 [0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[30;46m‚ôü[0m[0;42m [0m[0;46m [0m[0;42m [0m 6
5 [0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m 5
4 [0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[37;42m‚ôü[0m[37;46m‚ôü[0m[30;42m‚ôõ[0m 4
3 [0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m[0;42m [0m[0;46m [0m 3
2 [37;46m‚ôü[0m[37;42m‚ôü[0m[37;46m‚ôü[0m[37;42m‚ôü[0m[37;46m‚ôü[0m[0;42m [0m[0;46m [0m[37;42m‚ôü[0m 2
1 [37;42m‚ôú[0m[37;46m‚ôû[0m[37;42m‚ôù[0m[37;46m‚ôõ[0m[37;42m‚ôö[0m[37;46m‚ôù[0m[37;42m‚ôû[0m[37;46m‚ôú[0m 1

  ABCDEFGH

Eric Murphy won!
